ibis/backends/base/sql/compiler/select_builder.py

```diff
@@ -442,6 +445,28 @@ def _collect_Distinct(self, expr, toplevel=False):
 
         self._collect(expr.op().table, toplevel=toplevel)
 
+    def _collect_DropNa(self, expr, toplevel=False):
+        if toplevel:
+            op = expr.op()
+            if op.subset is None:
+                columns = [op.table[c] for c in op.table.columns]
```

Comment:
Yeah. Some backends (pandas, dask, spark) implement `dropna` directly, which is why there's a `DropNa` op I assume rather than just using `filter`. Wouldn't doing it in the api prevent/make it harder for those backends to call their native `dropna` methods?

Reply:
Yeah, that's a good point. At some point we'll revisit this, but for now I am okay with this implementation.